src/extension.ts

```diff
@@ -11,7 +11,9 @@ let client: LanguageClient
 let proc: cp.ChildProcess
 let logger: OutputChannel
 
-const versionRequirement = ">=1.10.1"
+const versionRequirement1 = ">=1.10.1"
```

Comment:
versionRequirement0 and versionRequirement1

Reply:
Let's make it a list where each i-th element is the version of the language server necessary, and the content of the element is the version requirement for Jolie.

Here: ">=1.10.1", ">=1.11.0"